flight/Modules/Autotune/autotune.c

```diff
@@ -207,25 +273,21 @@ static void AutotuneTask(void *parameters)
 			continue;
 		}
 
-		float throttle;
-
-		ManualControlCommandThrottleGet(&throttle);
-				
 		switch(state) {
 			case AT_INIT:
-
 				last_update_time = PIOS_Thread_Systime();
 
 				// Only start when armed and flying
-				if (flightStatus.Armed == FLIGHTSTATUS_ARMED_ARMED && throttle > 0) {
+				if (flightStatus.Armed == FLIGHTSTATUS_ARMED_ARMED) {
```

Comment:
The logic that was here was somewhat costly and in places incorrect.  If you were armed and in the middle of autotune (later in the code) it'd discard points when you have throttle=0 in the middle of the tune.... which you really shouldn't have and would render the tune completely untrustworthy.

Basically, what this block would do, is if you were armed and toggled to autotune when throttle is down, start the autotune 2 seconds after throttle up instead of 2 seconds after toggling to the mode.  IMO this is weird.